exercises/Style-R.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: exercise
+title: Good Style
+subtitle: Modify the Code!
+language: R
+---
+
+Programming computers to do cool science is a major advantage for modern scientists. But, developing research programs that are transparent, collaborative and reproducible is a major advantage for science. All data scientists can contribute to this goal by writing their code in easily readable, well described, and well commented scripts.
+
+This was not the case for the hack that wrote [this script]({{ site.baseurl }}/code/bad-style-R.R) to evaluate [tree composition in a Michigan swamp](http://www.esapubs.org/archive/ecol/E090/251/default.htm). *It works, but it's UGLY!* 
```

Comment:
Let's change the language here to be lest judgemental. Lots of folks in the class will have written ugly code and we don't want to make them feel bad about that in an unproductive way.

"This was not the case for the hack that wrote [this script]({{ site.baseurl }}/code/bad-style-R.R) to evaluate [tree composition in a Michigan swamp](http://www.esapubs.org/archive/ecol/E090/251/default.htm)"

-> 

"[This script]({{ site.baseurl }}/code/bad-style-R.R), which evaluates the [tree composition in a Michigan swamp](http://www.esapubs.org/archive/ecol/E090/251/default.htm), doesn't follow good style."

'*It works, but it's UGLY!*' -> '*It works, but it's difficult to understand exactly what it is doing.*'